src/net/sourceforge/kolmafia/utilities/NaiveSecureSocketLayer.java

```diff
@@ -79,11 +77,15 @@ public static void uninstall() {
   private static class NaiveTrustManager implements X509TrustManager {
     @Override
     public void checkClientTrusted(X509Certificate[] chain, String authType)
-        throws CertificateException {}
+        throws CertificateException {
+      throw new CertificateException("This is client unreal");
```

Comment:
What does this mean? I don't think I understand the error message. Same with the other

Reply:
I don't remember either.  I was thinking it didn't matter since they are never called in our code and exist only to pass static analysis.  But the message has been changed to something that should make sense if it were ever displayed.